grafana/dashboards/pai-clusterview-dashboard.json.template

```diff
@@ -0,0 +1,829 @@
+{"dashboard":{
```

Comment:
>dashboard" [](start = 2, length = 10)

all template should be put under service-deployment. otherwise it won't instantiate

Reply:
solved